tests/unittests/parsing/VisitorTests.cpp

```diff
@@ -250,6 +250,40 @@ endmodule
 )");
 }
 
+#include <iostream>
```

Comment:
What's the need for the iostream include here? It's a heavy header, I'd rather not include it if it's not absolutely necessary.

Reply:
it is leftover from debugging. Removed it